CMakeLists.txt

```diff
@@ -229,6 +229,8 @@ OCV_OPTION(WITH_NDSRVP "Use Andes RVP extension" (NOT CV_DISABLE_OPTIMIZATION)
   VISIBLE_IF RISCV)
 OCV_OPTION(WITH_HAL_RVV "Use HAL RVV optimizations" (NOT CV_DISABLE_OPTIMIZATION)
   VISIBLE_IF RISCV)
+OCV_OPTION(WITH_FASTCV "Use Qualcomm FastCV acceleration library for ARM platform" (NOT CV_DISABLE_OPTIMIZATION)
+  VISIBLE_IF (ARM OR AARCH64) AND ANDROID AND UNIX)
```

Comment:
Logical error `AND ANDROID AND UNIX`. Also Mac is UNIX.

Reply:
hi @asmorkalov , is this check correct?
`VISIBLE_IF (ARM OR AARCH64) AND (ANDROID OR (UNIX AND NOT APPLE AND NOT IOS AND NOT XROS)))`